schema/data/anchored/v0/classes/C06.jsonld

```diff
@@ -6,28 +6,32 @@
   },
   "schema:url": "https://www.registry.regen.network/crediting-protocols/ecometric---ghg-benefits-in-managed-crop-and-grassland-systems-credit-class",
   "schema:name": "GHG Benefits in Managed Crop and Grassland Systems Credit Class",
+  "schema:description": "This Credit Class and its supporting measuring and monitoring methodology, focuses on quantifying increases in soil organic carbon sequestration and storage in regenerative cropping and managed grassland agricultural systems.",
+  "rfs:hasPrimaryImpact": {
+    "rfs:hasImpactType": "INCREASED_CARBON_SEQUESTRATION_STORAGE",
+    "schema:name": "Soil Organic Carbon"
+  },
   "rfs:hasCoBenefits": [
     {
-      "rfs:impactType": "IMPROVED_SOIL_STRUCTURE",
+      "rfs:hasImpactType": "IMPROVED_SOIL_STRUCTURE",
       "schema:name": "Improved Soil Structure & Hydrological Properties"
     },
     {
-      "rfs:impactType": "REDUCED_FERTILIZER_AMENDMENTS",
+      "rfs:hasImpactType": "REDUCED_FERTILIZER_AMENDMENTS",
       "schema:name": "Reduced Fertilizer Usage"
     }
   ],
-  "schema:description": "This Credit Class and its supporting measuring and monitoring methodology, focuses on quantifying increases in soil organic carbon sequestration and storage in regenerative cropping and managed grassland agricultural systems.",
+  "rfs:managedUnderProgram": {
+    "schema:url": "'https://registry-program-guide.regen.network",
```

Comment:
```suggestion
    "schema:url": "https://registry-program-guide.regen.network",
```

Reply:
Fixed in b26f342a4a98290f321263f6b288b2a092a6b77e